kentikapi/users_integration_test.go

```diff
@@ -194,6 +195,108 @@ func TestClient_GetAllUsers(t *testing.T) {
 	}
 }
 
+func TestClient_GetUser_WithTimeout(t *testing.T) {
```

Comment:
It think that this test could be easily merged with TestClient_GetUser.

1. Provide serverHandlingDelay duration as test parameter and pass it to test HTTP handler.
2. Remove expectedReqs and just skip checking number of requests if expectedTimeout is true.

Please merge the tests or explain why they are separate.

Reply:
Please apply suggestion number 2.

Reply:
Sorry, I misread. On it.